tests/db_functions/comparison/test_cast.py

```diff
@@ -135,7 +135,12 @@ def test_expression_wrapped_with_parentheses_on_postgresql(self):
         The SQL for the Cast expression is wrapped with parentheses in case
         it's a complex expression.
         """
-        list(Author.objects.annotate(cast_float=Cast(models.Avg('age'), models.FloatField())))
+        Author.objects.create(name='Alice', age=24, alias='2')
+        average_age = Author.objects.aggregate(
+            cast_float=Cast(models.Avg('age'), models.FloatField())
+        )['cast_float']
+        self.assertIsInstance(average_age, float)
+        self.assertEqual(average_age, 12.5)
```

Comment:
This test in unrelated. Also, if we want to add these assertions I would recommend a separate test case.